packages/mds-api-helpers/index.ts

```diff
@@ -143,3 +143,20 @@ export const asJsonApiLinks = (req: express.Request, skip: number, take: number,
   }
   return undefined
 }
+
+export interface APIError {
+  error: string
+  error_description: string
+}
+
+class ServerError implements APIError {
+  error: string
+  error_description: string
+
+  constructor() { 
+    this.error = 'server_error'
+    this.error_description = 'an internal server error has occurred and been logged'
+   }
+}
+
+export const SERVER_ERROR = new ServerError()
```

Comment:
There are a bunch of `Error` classes in `mds-audit/src/exceptions.ts` (including `ServerError`)... now that we have `mds-api-helpers` perhaps those should be moved here and used throughout.

Reply:
yes to this